solutions/3724/01.py

```
@@ -0,0 +1,30 @@
+class Solution:
+    def minOperations(self, nums1: List[int], nums2: List[int]) -> int:
```

Ruff (0.14.7)

Undefined name `List`

(F821)